internal/action/bufpane.go

```diff
@@ -237,6 +238,8 @@ type BufPane struct {
 	// remember original location of a search in case the search is canceled
 	searchOrig buffer.Loc
 
+	// Used for start/end selection
+	StartSelectionLoc buffer.Loc
```

Comment:
Cursor already contains fields for storing start and end of selection... 0_o

Reply:
That one gets deselected on cursor move.